src/app.tsx

```diff
@@ -103,3 +112,21 @@ export class App extends React.Component<Props, State> {
         );
     }
 }
+
+const mapStateToProps = (state: GlobalState) => ({
+    id: state.id,
+    isOpptyLoading: state.isOpptyLoading,
+    isTouchSaving: state.isTouchSaving,
+    oppName: state.oppName,
+    touchCount: state.touchCount,
+});
+
+const mapDispatchToProps = (dispatch: any) => ({
```

Comment:
👍 I haven't been using this in QTR, but it's really nice. Need to go back and refactor